vllm/worker/habana_model_runner.py

```diff
@@ -16,6 +16,7 @@
 
 import habana_frameworks.torch as htorch
 import torch
+from neural_compressor.torch.quantization import finalize_calibration
```

Comment:
This brakes taking fp8 measurements with TP>1 - please revert changes made to this file